Sources/Plasma/PubUtilLib/plModifier/plResponderModifier.cpp

```diff
@@ -772,12 +772,11 @@ void plResponderModifier::ILog(uint32_t color, const char* format, ...)
     if (!format || *format == '\0')
         return;
 
-    const char* keyName = GetKeyName().c_str();
+    ST::string keyName = GetKeyName();
 
     // Make sure this key isn't in our list of keys to deny
-    for (int i = 0; i < gNoLogStrings.size(); i++)
-    {
-        if (strncmp(gNoLogStrings[i].c_str(), keyName, gNoLogStrings[i].length()) == 0)
+    for (const auto& it : gNoLogStrings) {
+        if (it.compare_i(keyName) == 0)
```

Comment:
This doesn't quite match the old code...  The old version checks that the `keyName` *starts with* `gNoLogStrings[i]` (`strncmp(..., gNoLogStrings[i].length())`), and doesn't compare case-insensitively